### ISSUE TITLE
LIMIT support for UPDATE statements

### ISSUE THREAD

**Issue by GoogleCodeExporter** [sagarswathi/h2database#362]
```
MySQL supports a UPDATE ... SET ... LIMIT syntax:

http://dev.mysql.com/doc/refman/5.6/en/update.html

This would be nice to add for compatibility.
```

Original issue reported on code.google.com by `arg...@gmail.com` on 2 Dec 2011 at 7:38

**Comment by GoogleCodeExporter** [comment]
```
Thanks a lot! I will add it to the roadmap.
```

Original comment by `thomas.t...@gmail.com` on 15 Dec 2011 at 10:02
- Changed state: **InRoadmap**

**Comment by GoogleCodeExporter** [comment]
```
I attached a patch which adds UPDATE LIMIT support and some unit tests.
```

Original comment by `arg...@gmail.com` on 22 Dec 2011 at 8:25

**Comment by GoogleCodeExporter** [comment]
```
I attached a new version of the patch.
```

Original comment by `arg...@gmail.com` on 22 Dec 2011 at 11:23

**Comment by GoogleCodeExporter** [comment]
Original comment by `thomas.t...@gmail.com` on 17 Feb 2012 at 1:59
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Should be fixed in H2 version 1.3.165.
```

Original comment by `thomas.t...@gmail.com` on 18 Mar 2012 at 5:37
- Changed state: **Fixed**
